mpact_api/mpact/models.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from django.db import models
+from django.utils import timezone
+
+
+# Create your models here.
+class UserData(models.Model):
+    id = models.CharField(primary_key=True, max_length=50)
+    is_bot = models.BooleanField()
+    first_name = models.CharField(max_length=100)
+    last_name = models.CharField(max_length=100, blank=True)
+    username = models.CharField(max_length=100, blank=True)
```

Comment:
`first_name` is required, but `username` is not? Are you sure?

Reply:
Yes, actually normal users will not set the username, as it is not required by telegram during registration.